database/wiiu/accept_friend_request_and_return_friend_info.go

```diff
@@ -73,6 +73,11 @@ func AcceptFriendRequestAndReturnFriendInfo(friendRequestID uint64) (*friends_wi
 		// * Offline
 		var lastOnlineTime uint64
 		row, err = database.Manager.QueryRow(`SELECT last_online FROM wiiu.user_data WHERE pid=$1`, senderPID)
+		if err != nil {
+			return nil, err
+		}
+
+		err = row.Scan(&senderPID, &recipientPID)
```

Comment:
This is my fault I fucked this up. This should scan into `lastOnlineTime`, not those 2